src/riak_kv_qry_queue.erl

```diff
@@ -67,18 +67,18 @@
 %%% API
 %%%===================================================================
 
--spec put_on_queue(pid(), [qry()], ?DDL{}) ->
+-spec put_on_queue(pid(), [qry()], ?DDL{}, riak_kv_qry_buffers:qbuf_ref() | undefined) ->
```

Comment:
When you see `undefined` during debugging you don't know if it is an intended or unintended value...